File: src/ch1/fizzbuzz.py
# 파이썬으로 FizzBuzz 문제를 해결하는 코드입니다.
for row in range(20):
    for col in range(5):
        number = row + col * 20 + 1
        if number % 3 == 0 and number % 5 == 0:
            text = 'FizzBuzz'
        elif number % 3 == 0:
            text = 'Fizz'
        elif number % 5 == 0:
            text = 'Buzz'
        else:
            text = str(number)

        print(f"{text:<12}", end="")
    print()

File: src/ch1/fizzbuzz.rs
fn main() {
    for row in 0..20 {
        for col in 0..5 {
            let number = row + col * 20 + 1;
            let text = if number % 3 == 0 && number % 5 == 0 {
                "FizzBuzz".to_string()
            } else if number % 3 == 0 {
                "Fizz".to_string()
            } else if number % 5 == 0 {
                "Buzz".to_string()
            } else {
                number.to_string()
            };

            // Print the number or text right-aligned in a 12-character wide field}
            print!("{:>12}", text);
        }
        println!();
    }
}